rasa/shared/nlu/training_data/formats/dialogflow.py

```diff
@@ -24,7 +24,9 @@
 
 
 class DialogflowReader(TrainingDataReader):
-    def read(self, fn: Text, **kwargs: Any) -> "TrainingData":
+    """Reader for NLU training data."""
+
+    def read(self, fn: Text, **kwargs: Any) -> "TrainingData":  # type: ignore[override]
```

Comment:
The error still persists, because it expects `Union[Text, Path]`

Reply:
does the code support both already? if not can minimal changes be done so that it supports both? (I went through similar changes on my PR, where I added support for `Path` without changing the code, just the type hints)

Reply:
Oh I changed this already, will request review now.